doc/dev/mfx_trace2-dev.md

```diff
@@ -0,0 +1,75 @@
+# MediaSDK tracing (mfx_trace2). Developer documentation
+
+## How to use mfx_trace2?
```

Comment:
No need in separate paragraph, add your See to the Overview as a first line.

Reply:
Done